### PR TITLE
Update main page highlight links

### PR DESCRIPTION
**Pull request by MaxPhilips**

### CODE REVIEW

**Comment by MaxPhilips** [comment]
+1 from Yegor internally

**Comment by MaxPhilips** [comment]
+1 from Jenni internally

File: layouts/highlights.html [hunk 1]
```diff
@@ -3,12 +3,13 @@
...
+      <p>New to the FHIR<sup>®</sup> standard? Take a look at Cerner's <a href="https://engineering.cerner.com/smart-on-fhir-tutorial/">SMART<sup>®</sup> on FHIR<sup>®</sup> tutorial</a>.</p>
+      <p>Browse different <a href="http://www.hl7.org/fhir/directory.cfml">versions</a> of the HL7<sup>®</sup> FHIR<sup>®</sup> standard. Cerner implements the <a href="http://hl7.org/fhir/DSTU2/">DSTU 2</a> and <a href="http://hl7.org/fhir/R4/">R4</a> versions of the HL7<sup>®</sup> FHIR<sup>®</sup> standard.</p>
     </li>
     <li class="highlight-module">
       <a href="http://wiki.hl7.org/index.php?title=Open_Source_FHIR_implementations"><span class="mega-octicon octicon-code"></span></a>
       <h2><a href="http://wiki.hl7.org/index.php?title=Open_Source_FHIR_implementations">Libraries</a></h2>
-      <p>Check out a number of <a href="http://wiki.hl7.org/index.php?title=Open_Source_FHIR_implementations">Open Source FHIR<sup>®</sup> Implementations</a>.</p>
+      <p>Check out a number of <a href="http://fhir.cerner.com/smart/#open-source-fhir-client-libraries">open source FHIR<sup>®</sup> implementations</a>.</p>
```
Comment (praneethk2094) [inline layouts/highlights.html]: Do we need to capitalize 'Open Source'?

Comment (MaxPhilips) [reply layouts/highlights.html]: no, I purposely lowercased it

**Pull request merged**